src/components/funFacts/FunFactComponent.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { funFacts, FunFact } from "./funfacts"
+import { useState } from "react"
+import "./FunFactPage.scss"
+
+interface FunFactsProps {
+    funFacts: FunFact[];
+}
+
+export const FunFactComponent: React.FunctionComponent<FunFactsProps> = ({ funFacts }) => {
+
+  const [selectedFact, setSelectedFact] = useState<number>(0);
+
+  function toggleFactDetail(funFactId:number){
+    if (selectedFact===funFactId){
+      setSelectedFact(0);
+    }
+    else {
+      setSelectedFact(funFactId);
+    }
+  }
+
+  return(
+    <div>
+      {funFacts.map( funFact => (
+        <div className="funfact">
+          <div className="funfact__prompt">
+            <h2 className="funfact__prompt--question">{ funFact.prompt }</h2>
```

Comment:
"Question" isn't a modifier here (I think) so much as it is a separate element
```suggestion
            <h2 className="funfact__prompt-question">{ funFact.prompt }</h2>
```

Reply:
ok thanks, yes i did wonder about this